docs/search.md

```diff
@@ -19,11 +19,6 @@ We have provided a Docker compose definition to run dependent services, includin
 ```sh
 docker compose run -p 8983:8983 solr
 ```
-If the above command does not work, you may want to try this instead:
```

Comment:
Same comment as above, let's standardise the language used, so "We have provided a Docker compose definition to run dependent services..." can be changed to something like "Docker Compose v2 is used to run dependent services..."